packages/react/src/components/Authenticator/FederatedIdentities/controls/types.ts

```diff
@@ -44,7 +41,7 @@ export interface CreateFederatedIdentitiesInput<
 
 export interface ProviderData<T extends string = string> {
   displayName: string;
-  icon: React.ReactNode;
+  icon?: FederatedProvider | React.ReactNode;
   providerName: T;
 }
```

Comment:
Not necessarily an issue but could a ProviderData passed in end up looking like the below?

```
{
  displayName: 'Amazon',
  icon: 'Amazon',
  providerName: 'Amazon'
}
```

I suppose they wouldnt need to do this since they could just pass in the Provider name instead of ProviderData instead, just funny

Reply:
Yep, they could do that. I wanted to allow a string icon, just in case they maybe wanted to change the displayName for a supported provider. There wouldn't be any way for the customer to pass in the supported icon unless they copy paste from our UI library.